lib/services/validator.dart

```diff
@@ -0,0 +1,22 @@
+class Validator{
+  static String? isFirstName(String firstname){
```

Comment:
If return a string value, function name should not start with 'is'. Change function name to 
static String? firstName(String firstname)

Do the same for other validators.